### ISSUE TITLE
.coveragerc doesn't support configuration of modules to be executed (like behave)

### ISSUE THREAD

**Issue by crd** [coveragepy/coveragepy#695]
Instead of specifying it all on the command line like this:

`$ coverage run --omit "config/*,features/steps/*" -m behave features/local/`

I would love to have something like this in my `.coveragerc` file:

```
[run]
omit = config/*,features/steps/*
module_commandline = behave features/local
```

Omit works, but there's no way (as far as I can tell) to specify I want to execute `behave` against the `features/local` directory in my example.

**Comment by nedbat** [comment]
I don't understand the advantage of this?  You'd still need to run "coverage run" somehow, so why not just leave the command line arguments there?

**Comment by merwok** [comment]
Tools like flake8 or pyflakes support having their settings and arguments in setup.cfg or tox.ini, which lets people run only `$ pytest` and have the right options always picked up.

**Comment by crd** [comment]
@nedbat `.coveragerc` supports some commandline options but not `-m` -- feels like it would be a natural extension. Do you disagree in general with the idea of having a coverage config file?

**Comment by nedbat** [comment]
@crd Of course I don't disagree with the idea of having a coverage config file.  It's full of all sorts of useful things.  I just hadn't personally seen the need for the rest of the "coverage run" command line to be in it, since it wouldn't save me a command line, just shorten it.  The config file was originally created as a way to control options when running coverage within a test runner, where I couldn't parse command-line options to control coverage's behavior.

**Comment by nedbat** [comment]
Implemented in https://github.com/nedbat/coveragepy/commit/e57549076220764ace6f2b67da3600ff5ae33f02.

**Comment by crd** [comment]
Thank you @nedbat, I realize you were iffy on the value of this line in .coveragerc but as someone on the hook for a few dozen prod apps it will be very nice to now be able to clone a project, `pipenv sync --dev`, followed by `pipenv run coverage run`

Appreciate it!

**Comment by nedbat** [comment]
This was released as part of 5.0a4.
